### ISSUE TITLE
[Question] Default Padding in run_net

### ISSUE THREAD

**Issue by mzouink** [MouseLand/cellpose#1218]
I am doing basic eval using CellposeSAM
input shape `(1, 3, 256, 256)`
And the run is basic, no 3d, no overlap

```python
masks, _, _ = model.eval(
        data,
        do_3D=False,
        diameter=None,
        channels=None,
        bsize=256,
        tile_overlap=0.0,
        batch_size=1,
        cellprob_threshold=0,
        normalize=False,
        interp= False,
    )
```

But still i found that the forward pass happens 4 times instead of 1
By checking the code i found the reason is this line:
https://github.com/MouseLand/cellpose/blob/79b0fcbf0296af8b021bcb812ff768d3e90aeb5d/cellpose/core.py#L195C45-L195C55

efault padding of 16 to x,y which generate more blocks and more forward. by removing it. things kept working so i am asking about the reason ?  

@carsen-stringer @mrariden
